product_docs/docs/pem/9/pem_architecture.mdx

```diff
@@ -101,12 +101,12 @@ The PEM client is a web-based application that runs in supported browsers. The c
 
 The client allows you to use PEM functionality that makes use of the data logged on the server through features such as dashboards, the Postgres Log Analysis Expert, and Capacity Manager.
 
-### SQL Profiler plugin
+### SQL Profiler extension
 
-You don't have to install the SQL Profiler plugin on every server, but you must install and configure the plugin on each server on which you want to use the SQL Profiler. You might also want to install and configure SQL Profiler on unmonitored development servers. You can also temporarily install the SQL Profiler plugin for ad hoc use.
+You don't have to install the SQL Profiler extension on every server, but you must install and enable the extension on each server on which you want to use SQL Profiler. You might also want to install and enable the SQL Profiler extension on unmonitored development servers. You can also temporarily install the SQL Profiler extension for ad hoc use.
 
-The plugin is installed with the EDB Postgres Advanced Server distribution but must be installed separately for use with PostgreSQL. The SQL Profiler installer is available from the [EDB website](https://www.enterprisedb.com/downloads/edb-postgres-enterprise-manager).
+The SQL Profiler installer is available from the [EDB website](https://www.enterprisedb.com/downloads/edb-postgres-enterprise-manager).
```

Comment:
I _think_ the original was more accurate - SQL Profiler is installed automatically with EPAS (or at least from the same installer for Windows). The Windows installers found at the link will be of no use to EPAS customers.

In both new and old versions, this sentence references an 'installer' which is only really relevant for Windows customers. In a weird inconsistency, the linked downloads page also gives no clue how to get SQL Profiler for Linux (for other products it provides a link to the repos page). We might need to run through this with @theadamwright and/or IT to figure out how we want this to be presented and explained, because I'm confused already!

Reply:
My understanding is that the SQL Profiler extension is not packaged with EPAS.